### ISSUE TITLE
The rootvalue function is not valid

### ISSUE THREAD

**Issue by caicailv** [cuth/postcss-pxtorem#83]
When I used version 6.0.0, an error occurred,
In the vite + Vue framework, I want to use two different proportions for the UI framework
But in the end, all the documents used the ratio of 50
This problem was resolved when I tried to switch to version 5.1.1
`
          postCssPxToRem({
            rootValue({ file }) {
              let num = file.includes('vant') ? 50 : 100
              return num
            },
            propList: ['*'],
          }),
`

**Comment by hemengke1997** [comment]
any reproduction?

**Comment by hemengke1997** [comment]
maybe you could try https://github.com/hemengke1997/postcss-pxtorem

**Comment by liejiayong** [comment]
Is there any solution?

**Comment by hemengke1997** [comment]
> Is there any solution?

yep! You can try this https://github.com/hemengke1997/postcss-pxtorem and let me know if you have any problems
